### DISCUSSION TITLE
Example of raw file.

### DISCUSSION THREAD

**Discussion by krk-krk-krk** [deepmodeling/deepmd-kit#1349] [Q&A]
"https://docs.deepmodeling.org/projects/deepmd/en/master/data/data-conv.html" requires box.raw, coord.raw, force.raw, energy.raw and virial.raw It is stated that.
However, these are in the files at "https://github.com/deepmodeling/deepmd-kit/tree/master/examples" and "http://www.deepmd.org/database/deeppot-se-data/" There are several things that are unclear how these file formats are written, as they do not contain raw files for. Specifically, I haven't found virial.raw yet.
Could you give me a sample of how to write virial.raw?

**Comment by amcadmus** [comment]
RTD https://github.com/deepmodeling/deepmd-kit/blob/master/doc/data/index.md

**Reply by krk-krk-krk** [reply]
I think # 337 is the answer I'm looking for. thank you.
